src/components/SponsorshipPackages/index.tsx

```diff
@@ -59,15 +59,7 @@ export interface SponsorshipPackagesProps {
   price?: Record<string, number>;
 }
 
-export const SponsorshipPackages = ({
```

Comment:
I would suggest rejecting this change, but include the type definition for the params of SponsorshipPackagesProps ?